website/docusaurus.config.js

```diff
@@ -63,11 +70,18 @@ const config = {
             label: 'Docs',
           },
           {
-            to: 'docs/api/modules/client',
+            type: 'doc',
+            docId: 'api/modules/client',
             activeBasePath: 'docs/api',
             position: 'left',
             label: 'API',
           },
+          // {
+          //   type: 'docsVersionDropdown',
+          //   position: 'right',
+          //   dropdownItemsAfter: [{ to: '/versions', label: 'All versions' }],
+          //   dropdownActiveClassDisabled: true,
+          // },
```

Comment:
Did you leave it here on purpose, @PaulLeCam ?

Reply:
Yes, that will be needed once we add a new version.